crates/iota-benchmark/src/bin/flamegraph.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[tokio::main]
+async fn main() {
+    use iota_test_transaction_builder::make_transfer_iota_transaction;
```

Comment:
what is this binary used for?
what do we do with that? use the json output and use it in grafana?

Reply:
Added a comment.